.github/ISSUE_TEMPLATE/oss_contribution.yml

```
@@ -0,0 +1,57 @@
+name: OSS Contribution
+description: Add or Modify OSS Information to OSORI
```

```suggestion
description: Add or Modify OSS Data Information to OSORI
```
What about modifying the description like above?

---

Done